apis/refs/v1beta1/computerefs.go

```diff
@@ -74,31 +74,31 @@ func ResolveComputeNetwork(ctx context.Context, reader client.Reader, src client
 		key.Namespace = src.GetNamespace()
 	}
 
-	computenetwork := &unstructured.Unstructured{}
-	computenetwork.SetGroupVersionKind(schema.GroupVersionKind{
+	computeNetwork := &unstructured.Unstructured{}
+	computeNetwork.SetGroupVersionKind(schema.GroupVersionKind{
```

Comment:
Nit: we might want to use the GVK from the API if we already have a dependency.  Not a big deal though.

Reply:
We don't have ComputeNetworkGVK yet, currently no Sci-fi migration plan for this resource. I'll leave it for now and update later.